src/reducers/inputs.js

```diff
@@ -21,8 +21,11 @@ export const setMyItems = (state, action) => {
   reduxPath.reduce((obj, key) => {
     level++
     if (level === reduxPath.length) {
-      if ((key in obj) != true || !Object.keys(obj[key]).includes('items')) {
-        obj[key] = { items: [] }
+      if (!(key in obj)) {
+        obj[key] = {}
+      }
+      if (!Object.keys(obj[key]).includes('items')) {
```

Comment:
Hmm, it may be JavaScript newbness, but it seems we're using two different strategies three lines apart to determine if an object contains a key. Would `if (!('items' in obj[key])) {` work here?